### PR TITLE
Spacecat 634 benchmarktest l bconn

### PR DESCRIPTION
**Pull request by fraugabel**

### CODE REVIEW

File: docs/TestCases.md [hunk 1]
```diff
@@ -133,4 +133,112 @@ In the target state, each test case described in the Bash script would be transl
...
\ No newline at end of file
+TODO list inputs for test cases in python
+
+### Errors and Updates
+
+For Openstack you can check ports and operational tools by using the command:
+`openstack catalog list`
+and will get:
+```
++-------------+----------------+---------------------------------------------------------+
```
Comment (piobig2871) [inline docs/TestCases.md:144]: please remove this table from documentation

File: features/steps/definitions.py [hunk 2]
```diff
@@ -45,6 +53,8 @@ def subnet_with_name_exists_in_network(
     ):
         network = context.client.network.find_network(name_or_id=network_name)
         assert network is not None, f"Network with name {network_name} does not exist"
+        # TODO: Verify by checking whether subnet is in network's subnet ids list
```
Comment (piobig2871) [inline features/steps/definitions.py:56]: please remove this comment

File: features/steps/definitions.py [hunk 3]
```diff
@@ -32,6 +32,14 @@ def when_a_router_with_name_exists(context, router_name: str):
     def connect_to_openstack(context, network_name: str):
         network = context.client.network.find_network(name_or_id=network_name)
         assert network is not None, f"Network with {network_name} doesn't exists"
+    
+    #TODO: loadbalancers
```
Comment (piobig2871) [inline features/steps/definitions.py:36]: comment to be removed

File: features/steps/definitions.py [hunk 4]
```diff
@@ -45,6 +53,8 @@ def subnet_with_name_exists_in_network(
     ):
         network = context.client.network.find_network(name_or_id=network_name)
         assert network is not None, f"Network with name {network_name} does not exist"
+        # TODO: Verify by checking whether subnet is in network's subnet ids list
+        # network.subnet_ids
```
Comment (piobig2871) [inline features/steps/definitions.py:57]: as above

File: features/steps/definitions.py [hunk 5]
```diff
@@ -115,6 +125,36 @@ def create_a_network(context, network_quantity: str):
             assert not context.client.network.find_network(
                 name_or_id=network), f"Network called {network} created"
 
+    #TODO: create loadbalancers inherit subnets and networks
```
Comment (piobig2871) [inline features/steps/definitions.py:128]: here as well

File: features/steps/definitions.py [hunk 6]
```diff
@@ -115,6 +125,36 @@ def create_a_network(context, network_quantity: str):
...
+    @then("I should be able to create {lb_quantity} loadbalancers for {subnet_name} in network {network_name}")
+    def create_a_lb(context, lb_quantity: str, subnet_name: str, network_name: str):
+        network = context.client.network.find_network(name_or_id=network_name)
+        assert network is not None, f"Network with name {network_name} does not exist"
+        print(network.id)
```
Comment (piobig2871) [inline features/steps/definitions.py:133]: remove print

File: features/steps/definitions.py [hunk 7]
```diff
@@ -115,6 +125,36 @@ def create_a_network(context, network_quantity: str):
...
+                # TODO: Verify by checking whether subnet is in network's subnet ids list
```
Comment (piobig2871) [inline features/steps/definitions.py:134]: and comments

File: docs/TestCases.md (hunk 1 above)
Comment (fraugabel) [reply docs/TestCases.md:144]: i understand

**Pull request closed without merging**